src/runtime/extended-access/gaa-metering.js

```diff
@@ -145,7 +145,11 @@ export class GaaMetering {
         unlockArticleIfGranted();
       } else if (showcaseEntitlement) {
         debugLog(showcaseEntitlement);
-        subscriptions.consumeShowcaseEntitlementJwt(showcaseEntitlement);
+        subscriptions.consumeShowcaseEntitlementJwt(showcaseEntitlement, () => {
```

Comment:
Could you move the definition of the `unlockArticle` variable here? The new code below looks like the first usage of the variable

Reply:
It's actually being used prior to this line in a `unlockArticleIfGranted`, so I've moved the definition of `unlockArticle` closest to where it's first used.

Reply:
Thanks! And sorry for missing that